packages/php/tests/ConfigTest.php

```diff
@@ -12,6 +12,8 @@ public function testConfigFileHasExpectedData(): void
             'api_key',
             'group_handler',
             'development_mode',
+            'denylist',
+            'allowlist',
             'blacklist',
             'whitelist',
```

Comment:
After you remove these from `config.dist.php`, this test will fail so these should get removed here too.

```suggestion
```